### PR TITLE
requested changes for select employees page

### PR DESCRIPTION
**Pull request by emmaacha**
Ticket IDs: 
- [Create Performance Evaluation Step 2A - Select Employees Screen#27](https://github.com/nerubia/kh360-react/issues/27#issuecomment-1794068970) 
- [Step 2A - no employees should be selected by default for new records#85](https://github.com/nerubia/kh360-react/issues/85)

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,3 +1,14 @@
...
-@tailwind utilities;
\ No newline at end of file
+@tailwind utilities;
+
+::-webkit-scrollbar {
+    -webkit-appearance: none;
+    width: 7px;
+  }
+  
+  ::-webkit-scrollbar-thumb {
```
Comment (jatnerubia) [inline src/App.css:10]: format document using prettier

File: src/features/admin/evaluation-administrations/[id]/select/SelectEmployeesTable.tsx [hunk 2]
```diff
@@ -15,22 +12,23 @@ export const SelectEmployeesTable = () => {
...
+  )
 
   const handleSelectAll = (checked: boolean) => {
+    let employeeIds = users.map((user) => user.id)
     if (checked) {
-      const employeeIds = allUsers.map((user) => user.id)
-      appDispatch(setSelectedEmployeeIds(employeeIds))
-      appDispatch(setCheckedAll(true))
+      appDispatch(
+        setSelectedEmployeeIds([...selectedEmployeeIds, ...employeeIds])
```
Comment (jatnerubia) [inline src/features/admin/evaluation-administrations/[id]/select/SelectEmployeesTable.tsx:23]: use previous data

File: src/features/admin/evaluation-administrations/[id]/select/SelectEmployeesTable.tsx [hunk 3]
```diff
@@ -15,22 +12,23 @@ export const SelectEmployeesTable = () => {
...
+      )
     } else {
-      appDispatch(setSelectedEmployeeIds(""))
+      employeeIds = users.map((user) => user.id)
+      appDispatch(
+        setSelectedEmployeeIds(
```
Comment (jatnerubia) [inline src/features/admin/evaluation-administrations/[id]/select/SelectEmployeesTable.tsx:28]: same

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a few changes ^^

File: src/features/admin/evaluation-administrations/[id]/select/SelectEmployeesTable.tsx (hunk 2 above)
Comment (emmaacha) [reply src/features/admin/evaluation-administrations/[id]/select/SelectEmployeesTable.tsx:23]: I tried po gamitin yung previous state kaso nageerror po sha
```
appDispatch(
        setSelectedEmployeeIds((prevSelectedEmployeeIds : number[]) => [...prevSelectedEmployeeIds, ...employeeIds])
      )
```
image

Comment (jatnerubia) [reply src/features/admin/evaluation-administrations/[id]/select/SelectEmployeesTable.tsx:23]: oh nvm, sa redux pala ito 😅

**Review by jatnerubia — APPROVED** [review]
Nice

**Review by Dev-Nino — APPROVED** [review]

**Pull request merged**